src/createServer.js

```diff
@@ -1,3 +1,67 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const caseOptions = ['SNAKE', 'KEBAB', 'UPPER', 'CAMEL', 'PASCAL'];
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const [originalText, queryString] = req.url
+      .split('?')
+      .map(query => query
+        .replace('/', ''));
+    const params = new URLSearchParams(queryString);
+    const targetCase = params.get('toCase');
+
+    const errors = [];
+
+    if (!originalText) {
+      errors.push({
+        message: 'Text to convert is required. '
+          + 'Correct request is: '
+          + '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    if (!targetCase) {
+      errors.push({
+        message: '"toCase" query param is required. '
+          + 'Correct request is: '
+          + '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    if (targetCase && !caseOptions.includes(targetCase)) {
+      errors.push({
+        message: 'This case is not supported. '
+          + 'Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+      });
+    }
```

Comment:
Let's split your logic, and create helper functions that will check errors.
Create a constant with error messages.
Also note:
- "+ 'Correct request is: ' + '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".'" - you do it twice
- Available cases are stored in `caseOptions`. Also, as it constant it should be in UPPER_CASE